### PR TITLE
Champ blocs

### PR DESCRIPTION
**Pull request by Marie59**
Initiate the PR for the tutorial for boulder fields biodiversity state analysis. The tuto is for 2 tools (IVR and dissimilarity).

### CODE REVIEW

File: topics/ecology/tutorials/champs-blocs/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+> The first time you use Galaxy, there will be no files in your history panel.
+{: .details}
+
+Now let's focus on our workflow on boulder field ecological state
+
+![Workflow for boulder field ecological state analysis](../../images/champbloc/workflow.png "Workflow")
+
+# Get data
+
+{% include _includes/cyoa-choices.html option1="Yes" option2="No" default="Yes" text="Are your ESTAMP data ready ?" %}
```
Comment (hexylena) [inline topics/ecology/tutorials/champs-blocs/tutorial.md:78]: very cool use of this!

**Comment by Marie59** [comment]
Sorry i left this training aside for sometimes ... I am trying to fix it but as it is my first training material I don't understand all the errors here. 
I get that the links for the tool are not correct but I don't see why. And i don't understand the other main error.
Can anyone help ?

File: topics/ecology/tutorials/champs-blocs/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+
+# Computing the Visual Boulder turning Indicator for each site
+
+Based on the proportion of "turned" and "not turned" blocs this indicator vary from 0 to 5 and can be quickly used.
+
+> <tip-title>How to get your tool ?</tip-title>
+>
+> > <hands-on-title>Use a tool</hands-on-title>
+> > 1.  click on for example the {% tool [IVR](toolshed.g2.bx.psu.edu/repos/ecology/cb_ivr) %} tool or type **ivr** in the tools panel search box (top left)
```
Comment (hexylena) [inline topics/ecology/tutorials/champs-blocs/tutorial.md:120]: Here it is missing a tool ID and tool version, this is just a toolshed repository owned by ecology, named cb_ivr, it doesn't list the specific tool inside that repository.

If you can get the full tool url (see the [guide on how to do it](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#how-to-find-these-ids)) that will fix it.

If you're using IUC standard tool repository setups, then it's probably

toolshed.g2.bx.psu.edu/repos/ecology/cb_ivr/cb_ivr/<some-version>, the `cb_ivr` duplication due to every tool getting placed into a separate tool repository on the toolshed

**Review by hexylena — COMMENTED** [review]
Hope this helps!

**Comment by Marie59** [comment]
Thanks a lot @hexylena ! I will check all that and try to fix everything !!

**Comment by Marie59** [comment]
Hiha ! everything worked !! Thank you @hexylena !!

**Comment by yvanlebras** [comment]
Youhouhou !!! Thnak you Marie !!

**Review by hexylena — APPROVED** [review]

**Pull request merged**